py3status/modules/xrandr.py

```diff
@@ -399,11 +402,15 @@ def xrandr(self):
         self._set_available_combinations()
         self._choose_what_to_display()
 
-        if self.fixed_width is True:
-            output = self._center(self.displayed)
+        if (len(self.available_combinations) < 2 and
+                self.hide_if_single_combination):
+            full_text = ''
```

Comment:
Hm, I still don't get it, you suggest that instead of creating `hide_if_single_combination` I define in my own config `format = '[{output}]'`, but with what adjustments in the code can the desired functionality be achieved? I hope not this kind of code? 😄

```
if (len(self.available_combinations) < 2 and '[' in self.format):
    output = ''
self.py3.safe_format(self.format, {'output': output})
```